manifests/metering-config/expose-route.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: metering.openshift.io/v1alpha1
+kind: Metering
+metadata:
+  name: "operator-metering"
+spec:
+  reporting-operator:
+    spec:
+      route:
+        enabled: true
+
+      authProxy:
+        enabled: true
+
+        # htpasswdData can contain htpasswd file contents for allowing auth
+        # using a static list of usernames and their password hashes.
+        #
+        # username is 'testuser' password is 'testpassword123'
+        # generated htpasswdData using: `htpasswd -nb -s testuser password123`
+        # htpasswdData: |
+        #   testuser:{SHA}y/2sYAj5yrQIN4TL0YdPdmGNKpc=
+        #
+        # change REPLACEME to the output of your htpasswd command
+        htpasswdData: |
+          REPLACEME
+
+        # cookieSeed is used to protect the cookie created if accessing the API
+        # via your browser.
+        #
+        # generate a 32 character random string using `pwgen 32`
```

Comment:
FYI, `pwgen` is not included in the default openshift installation as well as Mac/Windows.  I have a better luck using `openssl rand -base64 32` . Perhaps we can use that suggestion instead

Reply:
Feel free to open a PR

Reply:
ok, opened #497